data/hai/hai missions.txt

```diff
@@ -2099,3 +2099,213 @@ mission "Hai Pilgrimage"
 			`"Thank you, Captain, for making this journey. I apologize for not telling you more about me; the work I do is not something that I can reveal to any human who asks." He hands you <payment> and moves away. The surrounding Unfettered appear almost awed by his presence and consequentially give you a wide berth.`
 		log "Minor People" "Hareek" `An old Hai who claims to foresee the future. Dropped him off in Unfettered space so he could spread his message there. He appears to be well-respected amongst the Unfettered.`
+
+
+
+mission "Acorn Delights: Part 1"
+	minor
+	name "Acorns for Anoo"
+	description "Bring Eeeya to <destination> to deliver <cargo> to her missing granddaughter."
+	source "Allhome"
+	destination "Stonebreak"
+	passengers 1
+	cargo "acorn cookies" 3
+	to offer
+		random < 35
+		has "First Contact: Unfettered: offered"
+	on offer
+		conversation
+			`As you pass by a kitchen supply store in the spaceport, a dishevelled elderly Hai looks towards you with a face of deep concern. She waves you down.`
+			choice
+				`	(See what she wants.)`
+				`	(Ignore her.)`
+					decline
+			`	"My name is Eeeya, and I'm looking for my lost granddaughter, Anoo. She recently graduated from the medical school, and I was going to give her three tons of cookies, but she's gone! Please help me find her."`
+				goto "actual questions"
+			label questions
+			`	"So please, take me to find my granddaughter."`
+			label "actual questions"
+			choice
+				`	"Three tons of cookies!?"`
+					goto tons
+				`	"She's gone?"`
+					goto gone
+				`	"Why don't you ask the authorities for help?"`
+					goto authorities
+				`	"Can't your friends help you find her?"`
+					goto friends
+				`	"Let's go find her!"`
+					goto accept
+				`	"I have other places to be."`
+					decline
+			label tons
+			`	"Well, I... I don't know... I cook when I'm worried, and Anoo has been missing for three months! I've vacuum-sealed all three tons of cookies. They should last a year or two. Maybe she'll share them with her friends? She has many friends! Tons of friends! They all miss her, but none are willing to help...`
+				goto questions
+			label gone
+			`	"Anoo graduated from a medical academy on Frostmark four months ago, near the top of her class! A prominent hospital on the Hai-home needed a new trauma surgeon and said she would be perfect for the job. She accepted the position and said she would be on vacation for a few weeks.`
+			`	"A month later, she told them she won't be coming, and nobody has heard from her since.`
+				goto questions
+			label authorities
+			`	"I talked to the Central Medical Bureau, and they told me she got clearance for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got clearance. She has to register new prescriptions with the Bureau, so they should have heard from her.`
```

Comment:
```suggestion
			`	"I talked to the Central Medical Bureau, and they told me she got approval for practicing medicine anywhere in Hai space or beyond. They've heard nothing from her since she got approval. She has to register new medicines with the Bureau, so they should have heard from her.`
```